tools/releaseBuild/azureDevOps/templates/updateBuildTable.yml

```diff
@@ -0,0 +1,48 @@
+jobs:
+- job: updateBuildTable
+  displayName: Update Build Information Tbale
+  dependsOn:
+    ${{ parameters.parentJobs }}
+  
+  # the package EZ pools don't have the latest module needed to run this job
+  pool:
+    name: Hosted Mac Internal
```

Comment:
Any specific reason for this being macOS?

Reply:
Other than the comment, it's an internal pool